spec/factories/users.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user do
+    
+  end
```

Comment:
please fill this out, and test it out in the specs

Reply:
Question about this. . .this isn't in scope for what this feature test requires.  Do you still want spec tests on it?